src/app/modules/sidebar/sidebar.component.html

```diff
@@ -12,10 +12,12 @@
         <div class="user-info">
           <span class="user-name">{{player.getName()}}
           </span>
+<!--          
```

Comment:
Pleaser remove completely if it needs to be removed (-:

Reply:
deleted